Makefile

```diff
@@ -297,6 +297,5 @@ version-docs:
 	docker run --rm \
 		-v $(shell pwd)/docs:/docs \
 		-w /docs \
-		-u $(shell id -u):$(shell id -g) \
```

Comment:
is it possible to check if it rootless or otherwise, the files are created with root

Reply:
Added a shell script to check for rootless docker and set the uid/gid flags accordingly. Thanks for the heads up on this